src/components/Paginator/Paginator.jsx

```diff
@@ -113,10 +113,11 @@ const Paginator = createClass({
 		};
 	},
 
-	handleTextFieldChange(pageNum, totalPages) {
```

Comment:
Hey, while you're here, I actually made an oversight in omitting the event from the event handler, can you pass that along to `onSelect` in both places as well?

```
handleTextFieldChange(pageNum, event) {
...
return onPageSelect(selectedPageIndex, totalPages, event);
```